Analytics/Sources/Delivery/Analytics+Onboarding.swift

```diff
@@ -27,21 +27,33 @@ public extension Analytics {
         case photoConfirm
         case resume
         case start
+
+        /// Used for safewarding. Each onboarding step should be identified accordingly
```

Comment:
```suggestion
        /// Used for safeguarding. Each onboarding step should be identified accordingly
```
I think maybe you mean safeguarding?